racktables/racktables.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+from requests.auth import HTTPBasicAuth
+from bs4 import BeautifulSoup
+from os.path import join as join
+import requests
+import re
+
+class Racktables():
+  def __init__(self, url, username, password):
+    self.s = requests.Session()
+    self.s.verify = "/etc/ssl/certs/SUSE_Trust_Root.pem"
+    self.s.auth = HTTPBasicAuth(username, password)
+    self.url = url
+
+  def search(self, search_payload={}):
+    req = self.s.get(join(self.url, "index.php"), params="&".join("%s=%s" % (k,v) for k,v in search_payload.items()))
+    soup = BeautifulSoup(req.text, 'html.parser')
+    result_table = soup.find("table", {"class": "cooltable"})
+    result_objs = result_table.find_all("tr", lambda tag: tag != None) #racktables does not use table-heads so we have to filter the header out (it has absolutely no attributes)
+    return result_objs
+
+class RacktablesObject(object):
```

Comment:
```suggestion
class RacktablesObject:
```

This is not needed anymore in python3.